tools/depends/target/xz/Makefile

```diff
@@ -0,0 +1,37 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=xz
+VERSION=5.2.4
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.bz2
```

Comment:
Might be best to use gz (just uniformity with other depends really)  and any reason to not use 5.2.5?

Reply:
i basically just use the native xz as a template
https://github.com/xbmc/xbmc/blob/master/tools/depends/native/xz/Makefile
which uses 5.2.4 and .bz2

I'll update to 5.2.5 and .gz package :)

Reply:
Can stay with the bz2 then, do update to the 5.2.5 though

Reply:
Ping @wsnipex for an upload to mirrors for https://tukaani.org/xz/xz-5.2.5.tar.bz2

Reply:
you happy with it being called xz?
the native xz actually builds /src/xz/xz
where as this builds /src/liblzma/

Reply:
Fine with the name as xz

Reply:
uploaded to mirrors.

Reply:
not sure why the build failed. i suspect because I force pushed during the initial build?